README.md

```diff
@@ -233,28 +233,30 @@ You can also use key paths (for both dictionary keys and array indexes) to unbox
 
 ```json
 {
-    "name": "John",
-    "age": 27,
-    "activities": {
-        "running": {
-            "distance": 300
-        }
-    },
-    "devices": [
-        "Macbook Pro",
-        "iPhone",
-        "iPad"
-    ]
+	"name": "John",
```

Comment:
Yeah, suspected that, no worries :) Used to happen to me all the time too (I love iA Writer, use it for everything, even my shopping lists 😜), then I discovered that you can use spaces for indentation in iA Writer as well 🎉